### ISSUE TITLE
A11y_SQL Azure Data Studio_Extensions - install_Screen reader: After invoking "Install" button Screen Reader is not announcing "Installation" status whether it is installation is completed or not.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/azuredatastudio#24235]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG4.1.3; #ScreenReader; #Win32; #A11yWCAG2.1; #FTP; #STP; #Status Messages; #NVDA; #Narrator;

### Environment Details:
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11
Screen Reader: NVDA Version 2023.1(2023.1.0.27913)

### Repro Steps:

1. Launch Azure Data Studio.
2. From the left menu, Tab till the 'Extensions' icon and press enter to open the Extension Manager.
3. Choose any extension from the library.
4. Now Turn on NVDA/Narrator.
6. Tab till 'Install' button to install and press enter.
7. Now verify that NVDA/Narrator is not announcing installation is completed or not.

### Actual Results:
After invoking "Installation" button Screen Reader is not announcing "Installation" status whether it is installation is completed or not.
**Note:**
**1.Similar issue observed with NVDA**
**2.Similar issue is also reproducing in the below scenario:**
**1. Issue: Extensions - Uninstall**

1.Launch Azure Data Studio.
2. From the left menu, Tab till 'Extensions' icon and press enter to open the Extension Manager.
3. Choose any extension from the library.
4. Now Turn on NVDA/Narrator.
5.Tab till 'Install' button to install. and press enter. 'Uninstall' button should now appear.
6. Tab till 'Uninstall' button to uninstall. 
8. Now verify that NVDA/Narrator is not announcing "Uninstallation" is completed or not.

### Expected Results:
After invoking "Installation" button Screen Reader should announce "Installation" status whether it is installation is completed or not.

### User Impact:
Screen reader user will not be able to know installation status whether it is installation is completed or not. If NVDA/Narrator not announcing.

### Attachment:
[install and uninstall (1).webm](https://github.com/microsoft/azuredatastudio/assets/142479541/548a0d2f-b955-49b1-b312-eaf84d2e64c4)

[Install and Uninstall NVDA .webm](https://github.com/microsoft/azuredatastudio/assets/142479541/e6a47c1e-260b-424b-93c3-77cc83707fe1)

**Comment by kisantia** [comment]
@PadminiGariki this is the as the same first bug in https://github.com/microsoft/azuredatastudio/issues/24161 about the extension install status that has been fixed. The new bug that should have been opened is for the plus sign terminal button

**Comment by PadminiGariki** [comment]
@kisantia 
 **This issue is fixed** .After invoking "Plus" sign button Screen reader is announcing whether it is New Terminal created or not.
Attachments is added for reference. 

### Attachments:

[similar issue.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/6bfdac0a-2f85-4d7d-837f-ee9320cb2b46)

[NVDA similar issue (2).webm](https://github.com/microsoft/azuredatastudio/assets/142479541/81dd79e4-aeb4-4815-8e78-973e75d0247d)

**This issue is not fixed** .After invoking "Install" or "Uninstall" button Screen Reader is not announcing "Installation" or "Uninstallation" status whether it is (installed/uninstallation) completed or not. This is still repro.

Attachments is added for reference. 

### Attachments:

https://github.com/microsoft/azuredatastudio/assets/142479541/b791bf0e-6680-4d93-a7bf-200e5bbe051f

[Narrator issue not fixed.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/1b74e149-7dcf-4675-9d72-2782205b9947)

**Comment by kisantia** [comment]
@PadminiGariki please install the [latest insiders build](https://github.com/microsoft/azuredatastudio#download-the-latest-azure-data-studio-release) for testing this. The gear icon in the bottom left of the videos in your post has a "1" next to it, which means there is an update available.

In the latest insiders build, the installation completion is announced, as well as uninstalling:
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/196aa22f-d5d4-430b-b58a-59d4994d94c3)
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/9dde256b-6d51-4638-a8e0-a1b83e85d135)

**Comment by PadminiGariki** [comment]
@kisantia
verified the bug in this environment [Latest insiders build](https://github.com/microsoft/azuredatastudio#download-the-latest-azure-data-studio-release).
Here issue is partially fixed.
note issue is partially fixed but main issue is fixed. So, we create new bug for that note issue. hence the closed this bug.
for reference, please find the attachment.
#Closed; #Regressed:10-10-23;
### Attachments:

[Uninstalled not fixed Narrator.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/e8b32721-d7fe-4c1a-9ddd-ec104b2a48b0)

[Uninstalled not fixed NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/3d37a184-fc1c-42f7-927d-ff86f83ff2b1)

[installation Narrator fixed.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/ebbbc013-8bd3-41dc-9114-05e2880b674d)

[installation NVDA fixed.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/6896e982-3fe2-4592-9e40-7576ed2b69e7)

**Comment by PadminiGariki** [comment]
@kisantia 
As per Above comment. We create new bug for Note issue.
bug https://github.com/microsoft/azuredatastudio/issues/24629
